### DISCUSSION TITLE
Include markdown files in a "master" markdown file?

### DISCUSSION THREAD

**Discussion by mrichman** [jgm/pandoc#9622] [Q&A]
I want to have a master markdown file (e.g. `main.md`) where I keep the preamble and overall structure of a large doc. Each chapter or section would be stored in a secondary file (e.g. `01_foobar.md`) and could be included via some macro like `#include 01_foobar.md`. Is this easily doable?

**Comment by estedeahora** [comment]
I think you can use the [include-files filter](https://github.com/tarleb/lua-filters/tree/master/include-files).

**Comment by massifrg** [comment]
You may also try [pandoc-include-doc](https://github.com/massifrg/pandoc-include-doc).
